package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "homepage": "https://tsb.tecnico.ulisboa.pt/",
   "scripts": {
-    "start": "GENERATE_SOURCEMAP=false PORT=3005 react-scripts start",
-    "build": "GENERATE_SOURCEMAP=false react-scripts build",
+    "start": "set \"GENERATE_SOURCEMAP=false\" PORT=5000 && react-scripts start",
+    "build": "set \"GENERATE_SOURCEMAP=false\" && react-scripts build",
```

Comment:
Let's not do this.
I would prefer to, instead, do this cleaner solution: [https://stackoverflow.com/a/42232944/12771385](https://stackoverflow.com/a/42232944/12771385)
And update the `template.env`

Reply:
To be clear on this, revert these changes

Reply:
both port and GENERATE_SOURCEMAP=false are now in .env